docs/index.rst

```diff
@@ -0,0 +1,41 @@
+.. Karoo GP documentation master file, created by
+   sphinx-quickstart on Sun Oct  9 09:58:15 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Karoo GP
+===================
+Karoo GP is a Machine Learning application suite which provides both symbolic
+regression and classification data analysis. Written in the programming
+language Python, Karoo GP owes its foundation to the “Field Guide to Genetic
+Programming” by Poli, Langdon, McPhee, and Koza.
```

Comment:
```suggestion
language Python, Karoo GP owes its foundation to the "Field Guide to Genetic
Programming" by Poli, Langdon, McPhee, and Koza.
```
Better to keep ASCII quotes -- I think Sphinx will use fancy ones in the rendered output anyway.